src/data/connectors/index.ts

```diff
@@ -14,14 +14,23 @@ export * from "./measurement";
 export const Measurement = Mongoose.model<IMeasurementModel>("Measurement", MeasurementSchema);
 export const User = Mongoose.model<IUserModel>("User", UserSchema);
 
-const user = process.env.ROM_DB_USER;
-const pw = process.env.ROM_DB_PW;
+let ROM_DB_URL;
 
-export const MongooseConnection = Mongoose.connect("mongodb://" + user + ":" + pw + "@ds155737.mlab.com:55737/romdb")
+if (process.env.NODE_ENV === "prod") {
```

Comment:
I believe this could be omitted. Instead of using multiple environment variables, we could just use one.
When running on our local dev setups we set it to point on a local mongodb.
When running on, say Heroku, it is set to point to some production instance.
No need for code to handle this.